mender-convert

```diff
@@ -723,27 +723,41 @@ sudo true
 case "$1" in
   raw-disk-image-shrink-rootfs)
     do_raw_disk_image_shrink_rootfs || rc=$?
-  [[ $rc -ne 0 ]] && { return 1; }
+    [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
+    echo "Shrinked Linux image you can convert to Mender disk image is at "$raw_disk_image
     ;;
   raw-disk-image-create-partitions)
     do_raw_disk_image_create_partitions || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
```

Comment:
A new Mender disk image with partition layout to support Mender has been successfully created!
You can find the output Mender disk image at $mender_disk_image.